torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -202,6 +206,11 @@ bool isSupported(Node* node) {
       }
     }
 
+    if (FLAGS_torch_jit_disable_cat &&
+        node->matches("aten::cat(Tensor[] tensors, int dim=0) -> Tensor")) {
```

Comment:
Nit: all of the other uses of `cat` in this file use `aten::cat` as a symbol, and this uses aten::cat as a schema.